addon/index.js

```diff
@@ -0,0 +1,7 @@
+import ObjectController from './object_controller';
```

Comment:
`import ObjectController from "ember-legacy-controllers";`

instead of

`import ObjectController from "ember-legacy-controllers/object";`

i know it's minor but this package only has two exports :P